src/contracts/signer-identity.spec.ts

```diff
@@ -362,91 +332,80 @@ describe('signer-identity (identity based signer)', function test() {
       it('should perform fund transfer to contract with a fallback function, e.g. identities',
         async () => {
           const amountToSend = Math.floor(Math.random() * 1e3);
-          const balanceBefore = new BigNumber(await web3.eth.getBalance(signer.activeIdentity));
-          await executor.executeSend({
-            from: signer.underlyingAccount,
-            to: signer.activeIdentity,
+          const balanceBefore = new BigNumber(await web3.eth.getBalance(account0));
```

Comment:
```suggestion
          const balanceBefore = new BigNumber(await web3.eth.getBalance(identity0));
```

should this be identity?

Reply:
ignore it